modules/offload-s3/module.go

```diff
@@ -167,45 +168,54 @@ func (m *Module) Init(ctx context.Context,
 		m.DataPath = path
 	}
 
-	bucket := os.Getenv(s3Bucket)
-	if bucket != "" {
+	if bucket := os.Getenv(s3Bucket); bucket != "" {
 		m.Bucket = bucket
 	}
 
-	endpoint := os.Getenv(s3Endpoint)
-	if endpoint != "" {
+	var autoCreateBucket bool
+	if autoCreateBucketStr := os.Getenv(s3BucketAutoCreate); autoCreateBucketStr != "" {
+		v, err := strconv.ParseBool(autoCreateBucketStr)
```

Comment:
you can use `entcfg` package for this:
```
entcfg "github.com/weaviate/weaviate/entities/config"
```
there is a convenient method `entcfg.Enabled(os.Getenv(s3BucketAutoCreate))`